tests/test_default_headers.py

```diff
@@ -20,6 +24,25 @@ async def test_default_default_headers():
             assert response.headers["server"] == "uvicorn" and response.headers["date"]
 
 
+@pytest.mark.skipif(sys.platform == "darwin", reason="Unexpected behavior on MacOS.")
+@pytest.mark.anyio
+async def test_date_headers_update():
+    config = Config(app=app, loop="asyncio")
+    async with run_server(config):
+        async with httpx.AsyncClient() as client:
+            response = await client.get("http://127.0.0.1:8000")
+            date = response.headers["date"]
+            first_date = dt.datetime.strptime(date, "%a, %d %b %Y %H:%M:%S %Z")
```

Comment:
The `email.utils` function only works in one way :thinking:

Reply:
mm ok  sorry